### PR TITLE
fix(Jenkins CI): disable docker buildkit

### PR DESCRIPTION
**Pull request by RafalSkolasinski**

### CODE REVIEW

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-build-push | 923f4926e96f22a98212dd13f8757bf4723422a5 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/core-pr-4733-release-build-push-f8jzv) | `/test release`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@RafalSkolasinski: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
release-build-push | 923f4926e96f22a98212dd13f8757bf4723422a5 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/core-pr-4733-release-build-push-f8jzv) | `/test release`
notebooks | ae099202bdf4ed5ba49159a11ab3204e09940dd6 | [link](http://dashboard-jx.34.121.152.94.nip.io/namespaces/jx/pipelineruns/io-seldon-core-pr-4733-notebooks-bgtp6) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: testing/scripts/test
Comment (adriangonz) [inline testing/scripts/test:1]: Do you know why is this file required?

**Review by adriangonz — APPROVED** [review]

File: testing/scripts/test
Comment (RafalSkolasinski) [reply testing/scripts/test:1]: It was because I started here as dummy PR to just trigger CI. Now that tests passed and we want to merge it I gonna screenshot the test output and adjust PR to remove the empty test file and proceed with the merge :+1: 

Thanks for the reminder about it - I almost forgot :)

**Comment by RafalSkolasinski** [comment]
Tests pass

image
